modules/cluster/cloud-config.yml

```diff
@@ -103,11 +103,12 @@ write_files:
 
 runcmd:
 - |
+  yum install -y unzip
   yum install -y https://s3.dualstack.${region}.amazonaws.com/amazon-ssm-${region}/latest/linux_amd64/amazon-ssm-agent.rpm
-  curl -O https://bootstrap.pypa.io/get-pip.py
-  python get-pip.py --user
   export PATH=~/.local/bin:$PATH
-  pip install awscli --upgrade --user
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+  unzip -qq awscliv2.zip
+  ./aws/install -b /usr/bin
```

Comment:
Please remove the zip and unzipped folder after successful install. Otherwise it adds about 163MB extra to the image. For instance:
`rm -rf aws awscliv2.zip`

Reply:
Done